Generators/include/Generators/InteractionDiamondParam.h

```diff
@@ -0,0 +1,34 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \author R+Preghenella - October 2018
+
+#ifndef ALICEO2_EVENTGEN_INTERACTIONDIAMONDPARAM_H_
+#define ALICEO2_EVENTGEN_INTERACTIONDIAMONDPARAM_H_
+
+#include "SimConfig/ConfigurableParam.h"
+#include "SimConfig/ConfigurableParamHelper.h"
+
+namespace o2
+{
+namespace eventgen
+{
+
+struct InteractionDiamondParam : public o2::conf::ConfigurableParamHelper<InteractionDiamondParam> {
+  static constexpr int N = 3;
+  double mMean[N] = { 0., 0., 0. };  //[N]
+  double mSigma[N] = { 0., 0., 0. }; //[N]
+  O2ParamDef(InteractionDiamondParam, "InteractionDiamond");
```

Comment:
"InteractionDiamond" is quite long as key.

Reply:
this is true, although I believe it is better to have a long and clear name